moveit2.repos

```diff
@@ -23,6 +23,10 @@ repositories:
     type: git
     url: https://github.com/ros-controls/ros2_controllers
     version: master
+  ruckig:
+    type: git
+    url: https://github.com/pantor/ruckig.git
+    version: master
```

Comment:
Is this project stable enough to be in the main moveit2.repos file?

Reply:
We can use a fixed hash, as we did with ros2_control until not long ago.

Reply:
My main concern is that it hasn't been released yet and with each new version of ros we'll need to get the author's of this to release before we can release if moveit depends on it.  If we put this plugin in a separate repo it can still be included in tutorials and linked to by servo examples but it won't be a build/release dependency.

Reply:
OK. We can move this somewhere else, I guess. Then we also need to replace this plugin with a different plugin based on the lowpass filter that's currently in Servo so it has a default plugin to use.

Basically, what is left to do is...  (the list is getting long)
- Add functionality for parsing jerk and storing it (in the Joint Model, I think)
- Move this plugin somewhere else
- Delete the lowpass filter class from Servo and make it a plugin
- Refactor Servo to make it compatible with these plugins

Reply:
This is needed for now for CI to pass but after the next sync we can remove it as it was bloomed: https://github.com/ros/rosdistro/pulls?q=is%3Apr+Ruckig+is%3Aclosed